### PR TITLE
Add avalanche fuji support

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

File: docs/Developer Reference/link-token-contracts.md [hunk 1]
```diff
@@ -174,3 +174,20 @@ You can also <a href="https://www.youtube.com/watch?v=WKvIGkBWRUA" target="_blan
...
 |Symbol|LINK|
 |Decimals|18|
+
+## Avalanche
+
+### Fuji Testnet
+
+> 🚰Avax Faucet
+>
+> Free Avax for transactions: https://linkfaucet.protofire.io/fuji
```
Comment (PatrickAlphaC) [inline docs/Developer Reference/link-token-contracts.md:184]: Add fuji community faucet: https://linkfaucet.protofire.io/fuji

**Review by PatrickAlphaC — CHANGES_REQUESTED** [review]
Looks just about all set. 

I'm wondering if we can start adding oracles in here too. 
https://market.link/nodes/0cdf5cd6-b296-4230-a4dc-36f9b8323e28/metrics?network=1

File: docs/Developer Reference/link-token-contracts.md (hunk 1 above)
Comment (dmitrydao) [reply docs/Developer Reference/link-token-contracts.md:184]: > Looks just about all set.
> 
> I'm wondering if we can start adding oracles in here too.
> https://market.link/nodes/0cdf5cd6-b296-4230-a4dc-36f9b8323e28/metrics?network=1

I'm okay with that.

Comment (dmitrydao) [reply docs/Developer Reference/link-token-contracts.md:184]: Do you mean the link is a community faucet or you forgot to add another link?

**Review by StephenFluin — APPROVED** [review]

**Review by PatrickAlphaC — APPROVED** [review]

**Pull request merged**